.golangci.yaml

```diff
@@ -186,7 +186,7 @@ linters-settings:
   nakedret:
     # Make an issue if func has more lines of code than this setting, and it has naked returns.
     # Default: 30
-    max-func-lines: 0
```

Comment:
100 lines is kinda long. What do you need this for?

Reply:
For the precompile plugin’s Run function which is way more than the default of 30 lines. 100 is arbitrary though